proto/xds.proto

```diff
@@ -7,166 +21,172 @@ package envoy.service.discovery.v3;
 import "google/protobuf/any.proto";
 import "google/protobuf/duration.proto";
 
-option go_package="github.com/envoyproxy/go-control-plane";
+option go_package = "github.com/envoyproxy/go-control-plane";
 
 message Status {
-    // The status code, which should be an enum value of [google.rpc.Code][google.rpc.Code].
-    int32 code = 1;
-
-    // A developer-facing error message, which should be in English. Any
-    // user-facing error message should be localized and sent in the
-    // [google.rpc.Status.details][google.rpc.Status.details] field, or localized by the client.
-    string message = 2;
-
-    // A list of messages that carry the error details.  There is a common set of
-    // message types for APIs to use.
-    // google.protobuf.Any
-    repeated google.protobuf.Any details = 3;
+  // The status code, which should be an enum value of
```

Comment:
This shouldn't change...?

Reply:
Yeah, it is about proto format, let me revert.